website/client/js/controllers/partyCtrl.js

```diff
@@ -3,15 +3,20 @@
 habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','Challenges','$state','$compile','Analytics','Quests','Social', 'Pusher',
     function($rootScope, $scope, Groups, Chat, User, Challenges, $state, $compile, Analytics, Quests, Social, Pusher) {
 
+      var PARTY_LOADING_MESSAGES = 4;
+
       var user = User.user;
 
       $scope.type = 'party';
       $scope.text = window.env.t('party');
-      $scope.group = {loadingParty: true}
+      $scope.group = {loadingParty: true};
 
       $scope.inviteOrStartParty = Groups.inviteOrStartParty;
       $scope.loadWidgets = Social.loadWidgets;
 
+      // Random message between 1 and PARTY_LOADING_MESSAGES
+      $scope.partyLoadingMessage = Math.floor(Math.random() * PARTY_LOADING_MESSAGES) + 1;
```

Comment:
Agreed.  But I was going off of the example set by the [loading-screen tip code](https://github.com/HabitRPG/habitrpg/blob/29498976ee2e38a797f0ab0482b220a22c0b3008/website/server/controllers/top-level/pages.js#L27).